applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/hid_keyboard_leds_def_keyboard.h

```diff
@@ -15,13 +15,22 @@
  */
 const struct {} hid_keyboard_leds_def_include_once;
 
+/* For nRF54L15 SoC, only GPIO1 port can be used for PWM hardware peripheral output.
+ * Because of that, when using an nRF54L15 PDK, the following limitations must be considered:
+ *
+ * - For v0.2.x onboard LED1 cannot be used for PWM output.
+ * - For v0.3.0 onboard LED0 and LED2 cannot be used for PWM output.
+ *
+ * You can still use these LEDs with PWM LED driver, but LED color must be set either to
+ * LED_COLOR(255, 255, 255) or LED_COLOR(0, 0, 0) to avoid using PWM peripheral for the LEDs.
+ */
 static const struct led_effect keyboard_led_on = LED_EFFECT_LED_ON(LED_COLOR(255, 255, 255));
 static const struct led_effect keyboard_led_off = LED_EFFECT_LED_OFF();
 
 /* Map HID keyboard LEDs to application LED IDs. */
 static const uint8_t keyboard_led_map[] = {
 	[HID_KEYBOARD_LEDS_NUM_LOCK] = 2,
-	[HID_KEYBOARD_LEDS_CAPS_LOCK] = 3,
+	[HID_KEYBOARD_LEDS_CAPS_LOCK] = LED_UNAVAILABLE,
```

Comment:
Consider describing the LED limitations for the nRF Desktop app in the changelog for the CS release

Reply:
I wonder if we actually shouldn't add a generic note in a visible place to ensure customers are aware of the limitation (issue is SoC-specific, not app-specific). @nika-nordic, is the limitation already described somewhere so we could link there?

Reply:
That's irrelevant, we need to document sample/application differences for particular platforms if we have the documentation for the standard LED behaviour. It doesn't matter whether this HW bug, non-nRF Desktop SW bug or anything else.

We documented a workaround for Button 3/4 of the nRF54L15 PDK in all affected sample Readmes

Reply:
I think you can ask tech writers or UX team what we should do for this particular case

Reply:
Hi @annwoj, @divipillai
Do we already have a common doc note for this PWM limitation on nRF54L? Should we create a common place in doc describing nRF54L-specific limitations and link there from samples/apps or we will rather go with approach used previously for HW buttons limitations (copy-paste note describing board-specific/SoC-specific limitations)?

Reply:
We have only one common note regarding buttons in this section: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/nrf54l.html but it's in the dedicated place. All other limitations are mentioned in the cs release notes PR (https://github.com/nrfconnect/sdk-nrf/pull/14960). In the case described above I think we should choose the option that is going to be the most convenient for the user, which would be leaving a note in affected samples. However, in addition to that, I'd still mention it in the limitations